torchqdynamics/mesolve/mesolve.py

```diff
@@ -84,30 +86,37 @@ def mesolve(
     """
     # TODO H is assumed to be time-independent from here (temporary)
 
+    # default solver
+    if solver is None:
+        solver = Dopri45()
```

Comment:
The ordering of the different blocks of code (converting `H` to a tensor, then raising errors for `jump_ops`, etc...) was chosen to follow the arguments of the function, I think it's better to preserve it to ease the reading.